### PR TITLE
deprecate(profile): the profile/status view has been deprecated

### LINKED ISSUE #8697: Remove thewire post from top of user profile
_Maybe_ it could be moved to after all the other fields, but it should probably go away or an option should be added to remove it.

### PR DESCRIPTION
**Pull request by jdalsem**
fixes #8697

### CODE REVIEW

**Comment by hypeJunction** [comment]
Any reason?

**Comment by jdalsem** [comment]
yeah because of the issue... content is confusing and makes more sense in other locations.

File: mod/thewire/start.php [hunk 1]
```diff
@@ -38,9 +38,6 @@ function thewire_init() {
 	// Extend system CSS with our own styles, which are defined in the thewire/css view
 	elgg_extend_view('elgg.css', 'thewire/css');
 
-	// Add a user's latest wire post to profile
-	elgg_extend_view('profile/status', 'thewire/profile_status');
```
Comment (hypeJunction) [inline mod/thewire/start.php]: This not BC

**Review by hypeJunction — APPROVED** [review]

**Comment by hypeJunction** [comment]
Argh, didn't mean to approve.
I would say just wipe it in 3.0.

**Comment by hypeJunction** [comment]
I think having `profile/status` view makes sense, in 3.0, let's just remove thewire extension.

File: mod/thewire/start.php (hunk 1 above)
Comment (jdalsem) [inline mod/thewire/start.php]: it is... html is not part of BC policy...

**Comment by jdalsem** [comment]
i would rather rework the profile page in 3.0 a lot more than just removing the extension

File: mod/thewire/start.php (hunk 1 above)
Comment (mrclay) [inline mod/thewire/start.php]: This is a content change, not just differences in markup. master is the place to make this change.

**Review by mrclay — CHANGES_REQUESTED** [review]
Rather this just deprecate the view and it can go away in 3.

**Comment by mrclay** [comment]
We can decide fate of plugins elsewhere :)

**Pull request closed without merging**